source/extensions/filters/http/ext_proc/mutation_utils.cc

```diff
@@ -114,7 +114,7 @@ absl::Status MutationUtils::headerMutationResultCheck(const Http::HeaderMap& hea
 absl::Status MutationUtils::applyHeaderMutations(const HeaderMutation& mutation,
                                                  Http::HeaderMap& headers, bool replacing_message,
                                                  const Checker& checker,
-                                                 Counter& rejected_mutations) {
+                                                 Counter& rejected_mutations, absl::optional<bool> is_streamed_mode) {
```

Comment:
We can just name this parameterr as: bool send_body

Reply:
no, we don't need to set this for immediateResponse

Reply:
Would it be more readable to use `bool skip_content_length`? Conceptually the code cares about the addition of the content-length, not whether the mode is streaming or not.

Reply:
Agree to just make it bool skip_content_length.  Then you can pass in true or false directly. In the immediateResponse case, you can pass in true there.